cpp/src/parquet/file_writer.cc

```diff
@@ -23,6 +23,7 @@
 #include <utility>
 #include <vector>
 
+#include "arrow/util/endian.h"
```

Comment:
This is forgot previously?

Reply:
This is needed because I removed `#include "arrow/type.h"` from `properties.h`.